### PR TITLE
update: Alert feature added

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
I have created different alert types as Primary, Success, Warning, and Error alert

### CODE REVIEW

**Review by JantuDeb — CHANGES_REQUESTED** [review]
I would say that you create a CSS file for each component or branch. So, that it will be easy for us to review codes.

File: style.css [hunk 1]
```diff
@@ -66,326 +69,413 @@ body {
...
     background-color: var(--background-color);
-    color:var(--text-color);
-    width:25px;
+    color: var(--text-color);
+    width: 25px;
     text-align: center;
-    left:25px;
-    top:-10px;
+    left: 25px;
+    top: -10px;
```
Comment (anuj-kumary) [inline style.css:146]: One suggestion try to use one CSS unit everywhere in the project. It Maintains consistency in units.

Comment (Kothalkarpriya) [reply style.css:146]: I will implement it. Thank you

**Comment by Kothalkarpriya** [comment]
> I would say that you create a CSS file for each component or branch. So, that it will be easy for us to review codes.

Ok I will look into it

**Pull request merged**